Deeploy/MLIRAIETypes.py

```
@@ -0,0 +1,201 @@
+        operatorRepresentation : OperatorRepresentation
+            The parser's node representation (buffer names, sizes, types …).
+        **kwargs
+            Dialect-specific context provided by the deployer.
+        """
+        ...
+
+    # ------------------------------------------------------------------
+    # NodeTemplate overrides
+    # ------------------------------------------------------------------
+
+    def generate(self, operatorRepresentation = {}, **kwargs) -> str:
```

Ruff (0.15.9)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)